projects/kit/components/segmented/segmented.style.less

```diff
@@ -12,13 +12,13 @@ tui-segmented {
         --t-padding: 0.625rem;
         --t-gap: 0.5rem;
         --t-margin: -0.125rem;
+        --t-height: var(--tui-height-s);
```

Comment:
Are screenshots valid?

Reply:
No, my mistake. I'll fix it and address design changes in next commit